bin/render-activity-stream-html.js

```diff
@@ -93,6 +93,8 @@ for (const src of ${JSON.stringify(scripts, null, 2)}) {
   script.src = src;
 }
     </script>`;
+  // Hidden tag to check if prerendered version of html is used. See Bug 1516034
+  const isPrerenderedStatusTag = "<span hidden>Prerendered version html not shown</span>";
```

Comment:
Would you mind just making this a comment? I think it would be just as easy to inspect and no extra element

Reply:
+1, done